packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -1374,6 +1374,11 @@ export default class TradeStore extends BaseStore {
 
             if (has_currency_changed && should_reset_stake) {
                 obj_new_values.amount = obj_new_values.amount || getMinPayout(obj_new_values.currency ?? '');
+                if (this.is_dtrader_v2_enabled)
+                    this.setV2ParamsInitialValues({
```

Comment:
This is for fixing issue:
- Select non-crypto account (in my case it was demo account with stake 10 USD)
- Select crypto account  (in my case it was BTC)
- Check stake: it's shown previous stake (10 USD, 0.00016 BTC) NOT expected
- Click on Stake trade param. When Modal opens, stake field will have correct value (0.00016 BTC, not 10 USD)
- Select another crypto account (in my case it was ETH)
- Check Stake: it's shown previous stake (0.00016 BTC, not 0.004) NOT expected.
- Click on Stake trade param. When Modal opens, stake field will have correct value (0.004 BTC, not 0.00016 USD)

Sanity of V1 was added to qa-checklist

https://github.com/user-attachments/assets/16f948df-c012-4795-8a93-915f01f3b2b9